{"uuid": "b66eca37-e05e-43b5-97ed-190ef75ea810", "children": ["ba2f1723-c81c-4170-b576-4cda218014b2"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/test/Desktop/sprint_7/conftest.py\", line 22, in driver\n    driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\n"}, "start": 1728419183288, "stop": 1728419185872}], "start": 1728419183288, "stop": 1728419185929}